components/authz-service/server/v2/project.go

```diff
@@ -290,8 +291,10 @@ func rulesToProjectRules(rules []engine.Rule) []*api.ProjectRule {
 			Values: r.Values,
 		}
 	}
+	// TODO: The ProjectRule Type needs to be added to the database
 	rule := &api.ProjectRule{
 		Conditions: pr,
+		Type:       tags.RuleTypeNodeTag,
```

Comment:
This is hard coded to be of 'node' type. This will later have to be replaced with the type from the data store.